pkg/services/auth/jwt/key_sets.go

```diff
@@ -210,6 +262,16 @@ func (ks *keySetHTTP) getJWKS(ctx context.Context) (keySetJWKS, error) {
 		return jwks, err
 	}
 
+	if ks.bearerTokenPath != "" {
+		token, err := getBearerToken(ks.bearerTokenPath)
```

Comment:
Can we leave a comment here explaining why we might want to always read the bearer token from scratch? Might prevent someone from accidentally refactoring this out of existence.